### PR TITLE
release the asian manuscripts

### PR DESCRIPTION
**Pull request by gestchild**
## Who is this for?
People who want to see asian-manuscripts in the search results

ONLY MERGE ONCE  #5093 HAS BEEN MERGED

### CODE REVIEW

**Comment by jamesgorrie** [comment]
Are we reporting on any of these releases?
Seems like it would be nice?

**Comment by jamesgorrie** [comment]
"We've released 50000 more works" is good.
But "300000 views to works that were previously not viewed" feels better?

Ping @taceybadgerbrook @jennpb @harrisonpim

**Comment by jennpb** [comment]
Releases like these should be part of release notes, yes.

File: common/services/catalogue/api.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export function worksRouteToApiUrl(
   };
 }
 
-export const defaultWorkTypes = ['a', 'g', 'i', 'k', 'l', 'q'];
+export const defaultWorkTypes = ['a', 'g', 'i', 'k', 'l', 'q', 'b'];
```
Comment (davidpmccormick) [inline common/services/catalogue/api.js]: Make it alphabetical! You're an animal!

**Pull request merged**